application/commands/ResetPasswordCommand.php

```diff
@@ -21,8 +21,11 @@ public function run($sArgument)
         if (isset($sArgument) && isset($sArgument[0]) && isset($sArgument[1])) {
             $oUser = User::findByUsername($sArgument[0]);
             if ($oUser) {
+                Yii::import('application.helpers.common_helper', true);
```

Comment:
Needed ?

Reply:
OK , ` Error: Call to undefined function gT() in /home/usabilit/limesurvey/application/models/User.php:107`